### DISCUSSION TITLE
How do I remove monitor all?

### DISCUSSION THREAD

**Discussion by dmshimself** [tteck/Proxmox#1886] [Help Needed] [answered]
I've needed to turn monitor all off so I have stopped and disabled it, but 'something' keeps trying to restart it.  I've also tried masking it.  Is there a recommended way to remove it permanently?

**Comment by tteck** [comment, marked as answer]
Just follow the instructions.

**Comment by dmshimself** [comment]
Of course!  Many thanks for the reminder.  Old eyes!
